src/api.ts

```diff
@@ -1,6 +1,23 @@
 import { multinetApi } from 'multinet';
 import { host } from '@/environment';
+import { UserInfo } from '@/types';
 
 const api = multinetApi(`${host}/api`);
 
+export async function getUserInfo(): Promise<UserInfo | null> {
+  const resp = await fetch(`${host}/user/info`, {
+    credentials: 'include',
+  });
+
+  if (resp.ok) {
+    return resp.json();
+  }
```

Comment:
Currently I believe this endpoint returns a `403` if the user is not logged in, which will throw an error on the `await` call. We should either use a `try-catch` here, or wait until we change this on the server.

Reply:
Opened https://github.com/multinet-app/multinet-server/pull/393 to address this. If we merge that before this PR, then we should be good to go.

Reply:
https://github.com/multinet-app/multinet-server/pull/393 has been merged.